manifests/f/Freezer/Freezer/1.1.24/Freezer.Freezer.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Freezer.Freezer
+PackageVersion: 1.1.24
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+ProductCode: 6a8de3ec-77bc-5daa-a2f5-144fe44d3ebc
+Installers:
+- Architecture: x64
+  InstallerUrl: https://ia804505.us.archive.org/20/items/freezer-setup-1.1.24/Freezer%20Setup%201.1.24.exe
```

```suggestion
  InstallerUrl: https://archive.org/download/freezer-setup-1.1.24/Freezer%20Setup%201.1.24.exe
```

Here's the download link. It will redirect you to a CDN that will actually serve you the file. You can find it by right clicking `DOWNLOAD Setup x64` then clicking `Copy link address`.

---

I think the issue with this domain is that the administrators expect the Download URL to come from `https://freezerapk.com` but the publisher decided to upload their files to `https://archive.org` to potentially save up server costs.

`https://archive.org` is a third-party site, and even though the publisher uploaded their files there, it might have a high risk of removal or someone might be able to upload another Freezer installer with malicious code.